modules/eks/variables.tf

```diff
@@ -199,3 +199,41 @@ variable "tags" {
   description = "Deployment tags."
   default     = {}
 }
+
```

Comment:
Dont we also need to pass through the `namespace` variable.

Reply:
yes, adding it